### PR TITLE
Remove unnecessary shebangs

### PR DESCRIPTION
**Pull request by sfc-gh-turbaszek**
### Pre-review checklist
   * [ ] I've confirmed that instructions included in README.md are still correct after my changes in the codebase.
   * [ ] I've added new automated tests to verify correctness of my new code.
   * [ ] I've confirmed that my changes are up-to-date with the target branch.
   * [ ] I've described my changes in the section below.

### Changes description
...

### CODE REVIEW

**Review by sfc-gh-astus — APPROVED** [review]

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -4,7 +4,9 @@ on:
   pull_request:
     branches:
       - main
-
+  push:
+    branches:
+      - main
```
Comment (sfc-gh-astus) [inline .github/workflows/test.yaml:9]: sneaky change, I think it should be in separate PR

Comment (sfc-gh-turbaszek) [reply .github/workflows/test.yaml]: Removed
